pkg/bitcoin/chain.go

```diff
@@ -29,6 +29,14 @@ type Chain interface {
 	// returns an error.
 	GetBlockHeader(blockHeight uint) (*BlockHeader, error)
 
+	// GetTransactionMerkle gets the Merkle branch for a given transaction.
+	// The transaction's hash and the block the transaction was included in the
+	// blockchain need to be provided.
+	GetTransactionMerkle(
```

Comment:
What do you think about renaming it to `GetTransactionMerkleProof`?

Reply:
Good idea, done in 3ae0bf3c4dd91a1e0dffec6281c05066a3302616.